__tests__/endpoint.test.js

```diff
@@ -175,164 +175,38 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
-describe("POST /api/articles/:article_id/comments", () => {
-  test("Posts a new comment into an article with a given ID", () => {
-    const newComment = {
-      username: "rogersop",
-      body: "Hello this is my new comment",
-    };
-    return request(app)
-      .post("/api/articles/1/comments")
-      .send(newComment)
-      .expect(201)
-      .then(({ body }) => {
-        expect(body).toMatchObject({
-          comment_id: 19,
-          body: "Hello this is my new comment",
-          author: "rogersop",
-        });
-      });
-  });
 
-  test("404: If a new comment is posted with a username that doesn't exist an error will be returned", () => {
-    const newComment = {
-      username: "TomdaBomb",
-      body: "Hello this is my new comment",
-    };
+describe("DELETE comment by ID", () => {
+  test("responds with an empty object if no content after a successful deletion", () => {
     return request(app)
-      .post("/api/articles/1/comments")
-      .send(newComment)
-      .expect(404)
+      .delete("/api/comments/3")
+      .expect(204)
       .then(({ body }) => {
```

Comment:
Asserting the code is enough here as nothing should be returned from the endpoint in the event of a successful deletion